pkg/api/v1beta1/defaults.go

```diff
@@ -102,3 +108,14 @@ func init() {
 		},
 	)
 }
+
+// With host networking default all container ports to host ports.
+func defaultHostNetworkPorts(containers *[]Container) {
+	for i := range *containers {
+		for j := range (*containers)[i].Ports {
+			if (*containers)[i].Ports[j].ContainerPort == 0 {
+				(*containers)[i].Ports[j].ContainerPort = (*containers)[i].Ports[j].HostPort
```

Comment:
We changed it so that ContainerPort is not required if HostNetwork=true.

What I am trying to say is that, when a user defines ports in the HostNetwork=true scenario,
they still need to define which HostIP they plan to use. So, since they are specifying HostIP, i thought it was reasonable to expect that they specify HostPort as well.
ContainerPort can be considered optional in this case because, there is no container network namespace.